Updating Openshift documentation (#786)
DataDog/datadog-operator: docs/install-openshift.md

![Datadog Operator in the OperatorHub](assets/operatorhub.png)

> 3. Specify the namespace to install the Datadog Operator in, you can use `datadog` if you previously created the project or an existing one, e.g. `openshift-operators`:

Reviewer: ```suggestion
3. Specify the namespace to install the Datadog Operator in, you can use `datadog` if you previously created the project or an existing one, such as `openshift-operators`:
```

   ```
   Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your [Datadog API and application keys][3]

> **Note**: Starting with the version `1.0.3` of the Datadog Operator listing the Webhook Conversion is enabled by default. This will let you create DatadogAgent objects with the v1alpha1 or the new v2alpha1.

Reviewer: ```suggestion
**Note**: Starting with the version `1.0.3` of the Datadog Operator, listing the Webhook Conversion is enabled by default. This lets you create DatadogAgent objects with the v1alpha1 or the newer v2alpha1.
```
